engineering/definition-of-ready.md

```diff
@@ -51,16 +51,37 @@ should not be considered a feature we currently support!
 ### New hub
 
 (ready:config-change)=
-### Config change
+### Configuration change
 
-(ready:fault-fixing)=
-### Fault fixing
+This is usually about *modifying* a feature that was enabled via a
+[feature enablement](ready:feature-enablement) request earlier. So the definition of ready should
+be very similar.
+
+- [ ] Link to documentation about this feature, which has at least some mention of the possible
+      ways to make configuration changes. Since 2i2c documentation may not cover all the possible
+      options (as that would be too unweildy), this could be either a link to 2i2c documentation on
+      [infrastructure.2i2c.org](https://infrastructure.2i2c.org) or a link to upstream documentation
+      about various configuration options for this feature.
+- [ ] Any additional information needed for this specific configuration change.
+
+Functionally, this means that if we are requested to change config for something that we can't
+find any documentation for, we should
```

Comment:
Unfinished sentence

Reply:
Whoops. I completed this sentence, lmk what you think